index.ts

```diff
@@ -7,17 +7,19 @@ dayjs.extend(utc);
 
 export const eventify = (event: CalendarEvent) => {
   event.start = dayjs(event.start).toDate();
-  if (event.duration && event.duration.length && !event.end) {
-    const duration = Number(event.duration[0]);
-    const unit = event.duration[1];
-    event.end = dayjs(event.start)
-      .add(duration, unit)
-      .toDate();
-  }
-  if (event.allDay) {
-    event.end = dayjs(event.start)
-      .add(1, "day")
-      .toDate();
+  if (event.end == null) {
```

Comment:
That's just how I usually like to write 'falsyness' checks -- `undefined == null`, so this will also catch `undefined` values. (`== null` is really the only valid time to use the double-equals operator in JS...)

`!` on strings also actually has the weird behavior that `!'' === true`, which is (in my opinion) really strange & rarely what you want.

I'm happy to change to `!event.end` if that's the style you want to keep, it doesn't really make much of a difference in this case :) Let me know!